spec/actions_specs/crashlytics_spec.rb

```diff
@@ -55,13 +55,14 @@
               })
             end").runner.execute(:test)
 
-            expect(command).to eq([@crashlytics_bundle,
-                                   "wadus",
-                                   "secret",
-                                   "-ipaPath './fastlane/spec/fixtures/fastfiles/Fastfile1'",
-                                   "-notifications YES",
-                                   "-debug NO"
-                                  ])
+            expect(command).to eq([
+              @crashlytics_bundle,
```

Comment:
Use 2 spaces for indentation in an array, relative to the first position after the preceding left parenthesis.